rero_ils/modules/documents/api.py

```diff
@@ -45,6 +47,22 @@
 document_id_fetcher = partial(id_fetcher, provider=DocumentProvider)
 
 
+class DocumentsIndexer(IlsRecordIndexer):
+    """Indexing documents in Elasticsearch."""
+
+    def index(self, record):
+        """Index an document."""
+        return_value = super(DocumentsIndexer, self).index(record)
+        from ..persons.api import Person
```

Comment:
can you put this at the top of the file?

Reply:
Nop, circular inclusion!